app/ts/utils/constants.ts

```diff
@@ -72,8 +72,12 @@ export const FourByteExplanations = {
 	[get4Byte('mint(address)')]: 'Mint' as const,
 	[get4Byte('mint((address,address,uint24,int24,int24,uint256,uint256,uint256,uint256,address,uint256))')]: 'Mint' as const,
 	[get4Byte('burn(address)')]: 'Burn' as const,
-	[get4Byte('submitVote(uint256,bool)')]: 'Submit vote' as const,
-	[get4Byte('castVote(uint256,uint8)')]: 'Cast vote' as const,
+	[get4Byte('submitVote(uint256,bool)')]: 'Submit Vote' as const,
+	[get4Byte('castVote(uint256,uint8)')]: 'Cast Vote' as const,
+	[get4Byte('castVoteWithReason(uint256,uint8,string)')]: 'Cast Vote With Reason' as const,
+	[get4Byte('castVoteWithReasonAndParams(uint256,uint8,string,bytes)')]: 'Cast Vote With Reason And Params' as const,
+	[get4Byte('castVoteBySig(uint256,uint8,voter,bytes)')]: 'Cast Vote By Sig' as const,
+	[get4Byte('castVoteWithReasonAndParamsBySig(uint256,uint8,address,string,bytes,bytes)')]: 'Cast Vote With Reason And Params By Sig' as const,
```

Comment:
Turning these into something a regular person can easily understand is a challenge.  Below are a few minor tweaks that may help a little bit, but still leave quite a bit to be desired.
```suggestion
	[get4Byte('castVoteWithReason(uint256,uint8,string)')]: 'Cast Vote with Reason' as const,
	[get4Byte('castVoteWithReasonAndParams(uint256,uint8,string,bytes)')]: 'Cast Vote with Reason and Additional Info' as const,
	[get4Byte('castVoteBySig(uint256,uint8,voter,bytes)')]: 'Cast Vote by Signature' as const,
	[get4Byte('castVoteWithReasonAndParamsBySig(uint256,uint8,address,string,bytes,bytes)')]: 'Cast Vote with Reason And Additional Info by Signature' as const,
```